packages/image_picker/image_picker_ios/pigeons/messages.dart

```diff
@@ -35,9 +35,9 @@ class SourceSpecification {
 @HostApi(dartHostTestHandler: 'TestHostImagePickerApi')
 abstract class ImagePickerApi {
   @async
-  @ObjCSelector('pickImageWithSource:maxSize:quality:')
-  String? pickImage(
-      SourceSpecification source, MaxSize maxSize, int? imageQuality);
+  @ObjCSelector('pickImageWithSource:maxSize:quality:requestFullMetadata:')
```

Comment:
Nit: this param would be more idiomatic in ObjC as just `fullMetadata:`; the "with" is implied to go with following arguments, and "with full metadata" reads much better than "with request full metadata".